tools/MGnify_search.md

```diff
@@ -1,31 +1,101 @@
 
-## Input / Prerequisites
-* Access to [MGnify Search](https://www.ebi.ac.uk/metagenomics/sequence-search/search/phmmer)
+
+**Tutorial Objective**: This tutorial will cover both types of searches using two different inputs. In the `Text Search`, a study investigating the [Impact of diet gene expression in bovine rumen microbiomes](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJEB7104), specifically a run where an obelisk was found, will be used while `Sequence Search` will be used against a full length sequence of Arginine deiminase from _Streptococcus sanguinis_ derived from MGnify example data.
+
+## MGnify Text Search
+
+### Input / Prerequisites
+- Access to [MGnify Text Search](https://www.ebi.ac.uk/metagenomics/search/studies)
+- Study Details:
+  - bioProject: `PRJEB7104`
+  - SRA run: `ERR747931`
+
+### Output
+
+Given a search term, MGnify will output results showing those projects or samples with metadata containing that term. Each result is indexed by a MGnify identifier prefixed with `MGY`, a letter corresponding to the data type (`S` for study and `A`) for analysis and a unique eight-digit number.
+
+#### 1. Navigate to `MGnify Text Search`
+
+#### 2. Performing the Search
+
+On the text box, enter the bioProject accession: `PRJEB7104` then click ***Search***
+
+#### 3. Results Table
+
+In the results table, you should see the bioProject accession under `ENA accession` and its corresponding `MGnify ID`. Click this id and you will be redirected to the overview page for this study.
+
+![Results Table](~/img/MGnify_search/bioProject.png "Figure 1: Results Table")
+
+#### 4. Study Overview on `MGnify`
+
+On the overview page of the study, there are four sections:
+- `External Links` - a link to the project submission on ENA
+- `Classification` - how the study is classified within `MGnify`. In this case, it is classified as `root:Host-associated:Mammals:Digestive system:Large intestine:Fecal`.
+- `Description` - title of the study and where it was conducted
+- `Programmatic Access` - links on analyzing the data via R or Python
+- `Analyses` - a table of all the samples/runs associated with this project.
+
+#### 5. In the `Analyses` table, locate the sample with the SRA accession `ERR747931` under the `Run/Assembly accession` column. Click its `Analysis accession` code prefixed with `MGYA`. 
+![Analyses Table](~/img/MGnify_search/analysisTable.png "Figure 2: Analyses Table")
+
+#### 6. Run Analysis
+
+In the analysis page of the run, there are six tabs containing the following:
+- `Overview` - description of the study, sample, run, and pipeline used as well as experimental details.
+
+![Analysis Overview](~/img/MGnify_search/analysisOverview.png "Figure 3: Analysis Overview Page")
```

Comment:
The `~` tilde here is not working, GitHub does not recognize it as shortcut to `PWD`, can you amend and remove the tildes in the image paths. I believe that should resolve the image loading.